search/templates/search/base.html

```diff
@@ -47,7 +47,7 @@ <h1 class="title is-clearfix">{% block title %}Search{% endblock title %}</h1>
       <button class="button is-small" id="feedback-button">Feedback?</button>
     </div>
   </div>
-    <content>
+    <div class="content">
```

Comment:
sad to see HTML5 rejected the semantic elements: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/content

Learned from a PR though! :tada: